shipctl/x86_64/Ubuntu_16.04/utility.sh

```diff
@@ -1275,6 +1322,59 @@ _notify_newrelic() {
   fi
 }
 
+_notify_airbrake() {
+  local curl_auth=""
+  local r_obj_type=""
+  local r_project_id="${opt_project_id}"
+  local r_endpoint=$(get_integration_resource_field "$r_name" url)
+  local r_token=$(get_integration_resource_field "$r_name" token)
+  local default_airbrake_payload="{\"environment\":\"\${opt_environment}\",\"username\":\"\${opt_username}\",\"email\":\"\${opt_email}\",\"repository\":\"\${opt_repository}\",\"revision\":\"\${opt_revision}\",\"version\":\"\${opt_version}\"}"
+
+  if [ -z "$opt_type" ]; then
+    echo "Error: --type is missing in shipctl notify"
+    exit 99
+  fi
+  if [ "$opt_type" == "deploy" ]; then
```

Comment:
how come one is singular and one is plural?

Reply:
`--type` argument uses singular like `--type=deploy` since shipctl notify tries to post only one deploy object to airbrake per invocation.

The API endpoint for posting deploy object looks like `POST https://airbrake.io/api/v4/projects/PROJECT_ID/deploys?key=PROJECT_KEY` => Here `deploys` is used in plural form.

Hence the difference.